src/.idea/misc.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should not be included in the commit. I wonder why it is not automatically excluded by the gitignore.